apps/comps/components/competition-card.tsx

```diff
@@ -104,7 +105,15 @@ export const CompetitionCard: React.FC<CompetitionCardProps> = ({
               <h3 className="text-secondary-foreground mb-1 text-xs font-semibold uppercase">
                 Reward
               </h3>
-              {competition.rewards ? (
+              {competition.rewardsTge ? (
```

Comment:
we first check if `rewardsTge` is available, if not we check `rewards`, if not we display `TBA`